docs/performance.rst

```diff
@@ -34,7 +34,7 @@ Here are a few hints how to get most out of ``structlog`` in production:
 
       configure(cache_logger_on_first_use=True)
 
-   This has the only drawback is that later calls on :func:`~structlog.configure` don't have any effect on already cached loggers -- that shouldn't matter outside of testing though.
+   This has the only drawback is that later calls on :func:`~structlog.configure` don't have any effect on already cached loggers -- that shouldn't matter outside of :ref:`testing <testing>` though.
```

Comment:
I think this has to be `doc`, not ref. At least that's what I'm getting from the CI's complaining. :)

Reply:
(you may even get away with just backticks because we have `default_role = "any"` set.)